README.md

```diff
@@ -1 +1,43 @@
-# Microverse-Capstone-Concert
\ No newline at end of file
+# Microverse-Capstone-Concert
+
+> This is a Microverse capstone project to design a concert website template from behance.  
+
+## Built With
+
+- HTML 
+- CSS
+- Javascript
+
+## Link to Online Version
+
+- https://enning94.github.io/Concert-Website-Microverse-Capstone/
```

Comment:
- [ ] Kindly ensure your live demo is working, all the sections and pages should be working, currently, I am getting something different when I click on your link:

![image](https://user-images.githubusercontent.com/61408860/181727538-53502627-7e9d-4510-a4c5-a2afb3297eed.png)